app/assets/stylesheets/components/_button-link.scss

```diff
@@ -0,0 +1,16 @@
+.button-link {
+  &#{&} {
+    cursor: pointer;
+    display: block;
+    line-height: 1.5;
+  }
+
+  &#{&}:hover,
+  &#{&}:active {
+    text-decoration: none;
+  }
+
+  &#{&}:focus {
+    outline: none;
+  }
```

Comment:
Is the `&#{&}` to increase specificity of the selector to override other styles?

It might be clearer to trace back if we target the other selectors we're trying to override:

```suggestion
  cursor: pointer;
  
  &.usa-button--unstyled {
    display: block;
    
    &:hover,
    &:active {
      text-decoration: none;
    }
    
    &:focus {
      outline: none;
    }
  }
```

Reply:
fixed in 590b47583e161ed1b0381f01cb91d48c37941da7